### PR TITLE
847609: Need to update UG documentation in HTMLTOPDF section

### PR DESCRIPTION
**Pull request by Sri-hari-haran-g**
Hi All,

I have updated UG documentation in HTMLTOPDF section. Kindly review this.

Regards,
Srihariharan Gopalakrishnan

### CODE REVIEW

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 1]
```diff
@@ -8,50 +8,33 @@ documentation: UG
...
 <br/><br/>
-Place the BlinkBinaries folder in bin folder of the project. 
+you can set BlinkPath if you need to set the Path explicitly
 <br/><br/>
-The BlinkBinaries will be available in the HTMLConverter installed location <span style="color:gray;font-size:14px"><i>($SystemDrive\Program Files (x86)\Syncfusion\HTMLConverter\xx.x.x.xx\BlinkBinaries)</i> </span>
-<br><br/>
-The BlinkBinaries also available in NuGet package installed location if you are using Blink HTML converter from NuGet packages. 
-
-</td>
+Ex path: <i>D:\HTMl-to-PDF\HTMl-to-PDF\bin\Debug\net7.0\runtimes\win-x64\native</i>
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:35]: Add code example here

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 2]
```diff
@@ -8,50 +8,33 @@ documentation: UG
...
+Make sure that after installing the NuGet package and building, the 'runtimes' folder will be copied automatically.
+<br/><br/>
+Please refer the below screenshot,
+<br/><br/>
+<img src="htmlconversion_images/runtime_folder.png">
 <br/><br/>
 (Or)
 <br/><br/>
-Place the BlinkBinaries folder in bin folder of the project. 
+you can set BlinkPath if you need to set the Path explicitly
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:33]: If you require explicit path setting, you can establish BlinkPath.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 3]
```diff
@@ -8,50 +8,33 @@ documentation: UG
...
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occurs if the '<i>runtimes</i>' folder doesn't get copied properly from the NuGet folder.
 </td>
 </tr>
 <tr>
 <th style="font-size:14px" width="100px">Solution</th>
 <td>
-Set the path of the BlinkBinaries folder to the BlinkPath property of BlinkConverterSettings.
+Make sure that after installing the NuGet package and building, the 'runtimes' folder will be copied automatically.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:25]: Ensure that upon NuGet package installation and building, the 'runtimes' folder is automatically copied.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 4]
```diff
@@ -8,50 +8,33 @@ documentation: UG
...
 
 <table>
 <th style="font-size:14px" width="100px">Exception</th>
-<th style="font-size:14px">BlinkBinaries are missing</th>
+<th style="font-size:14px">Blink files are missing</th>
 <tr>
 <th style="font-size:14px" width="100px">Reason
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occurs if the '<i>runtimes</i>' folder doesn't get copied properly from the NuGet folder.
```
Comment (SowmiyaLoganathan) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:19]: The exception may occur if the '<i>runtimes</i>' folder isn't copied correctly from the NuGet folder.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 5]
```diff
@@ -8,50 +8,52 @@ documentation: UG
...
-<tr>
-<th style="font-size:14px" width="100px">Mapping BlinkBinaries in Web Application</th>
-<td>
-{% highlight c# tabtitle="C#" %}
-
-//To refer BlinkBinaries from project location
-settings.BlinkPath = Server.MapPath("BlinkBinaries");
-
-//or
+Ex path: <i>C:\HtmlConversion\HTMl-to-PDF\HTMl-to-PDF\bin\Debug\net7.0\runtimes\win-x64\native\BlinkBinaries</i>
```
Comment (PrakashViswanath) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:35]: C:\HtmlConversion\HTMl-to-PDF\HTMl-to-PDF\bin\Debug\net7.0\runtimes\win-x64\native\

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 6]
```diff
@@ -8,50 +8,52 @@ documentation: UG
...
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occur if the <i>'runtimes'</i> folder is not copied correctly from the NuGet folder.
 </td>
 </tr>
 <tr>
 <th style="font-size:14px" width="100px">Solution</th>
 <td>
-Set the path of the BlinkBinaries folder to the BlinkPath property of BlinkConverterSettings.
+Ensure that upon NuGet package installation and building, the 'runtimes' folder is automatically copied.
```
Comment (PrakashViswanath) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:25]: Ensure that the runtimes folder is copied properly to bin folder of the application from NuGet package location.

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 7]
```diff
@@ -8,50 +8,52 @@ documentation: UG
...
+<br/><br/>
+Please refer to the below screenshot,
+<br/><br/>
+<img src="htmlconversion_images/runtime_folder.png">
 <br/><br/>
 (Or)
 <br/><br/>
-Place the BlinkBinaries folder in bin folder of the project. 
+If you require explicit path setting, you can establish BlinkPath.
```
Comment (PrakashViswanath) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:33]: You can set the runtimes folder path explicitly in BlinkPath property in BlinkConverterSettings class.

**Review by PrakashViswanath — CHANGES_REQUESTED** [review]

**Pull request merged**